.gitignore

```diff
@@ -11,6 +11,7 @@
 /*.pem
 /keys
 /roles
+/sbkeys/**/
```

Comment:
This excludes subdirectories of `sbkeys` since that is where the default "local" profile is written. It doesn't affect files in the `sbkeys` directory.